workgroup/src/main/java/software/amazon/athena/workgroup/Translator.java

```diff
@@ -22,7 +22,7 @@
 
   WorkGroupConfiguration createSdkWorkgroupConfigurationFromCfnConfiguration(software.amazon.athena.workgroup.WorkGroupConfiguration cfnConfiguration) {
     return WorkGroupConfiguration.builder()
-      .bytesScannedCutoffPerQuery(cfnConfiguration.getBytesScannedCutoffPerQuery() != null ? cfnConfiguration.getBytesScannedCutoffPerQuery().longValue() : null)
```

Comment:
so we don't need to handle null case anymore?

Reply:
I don't think this code was handling null case, it was just unnecessarily boxing. 
`x != null ? x.longValue() : null` will return a `long` or `null`, my change makes it return `Long` (which may be `null`). Since workgroup configuration has `Long` as type for bytes scanned now, we can just pass it directly